src/commands/subcommands/community/handleSuggest.ts

```diff
@@ -37,7 +37,7 @@ export const handleSuggest: CommandHandler = async (
 
     const suggestionChannel = guild.channels.cache.find(
       (el) => el.id === config.suggestion_channel
-    ) as TextChannel;
+    ) as TextChannel || ForumChannel ;
```

Comment:
```suggestion
    ) as TextChannel | ForumChannel ;
```

This is going to open up a whole can of worms when you commit it - but `||` is the logical OR, `|` is the type operator.

Reply:
Got it... do you want me to leave it or change it?

Reply:
@naomi-lgbt The compiler only likes a single `|`